python/src/usecases/chaboche_model.py

```diff
@@ -9,7 +9,7 @@
 
 def g(X):
     strain, R, C, gamma = X
-    stress = R + C * (1 - np.exp(-gamma * strain)) / gamma
+    stress = R + C * (1.0 - np.exp(-gamma * strain)) / gamma
```

Comment:
Maybe np.expm1 should be preferred here?

Reply:
```suggestion
    stress = R - C * np.expm1(-gamma * strain) / gamma
```

Reply:
This is technically OK, but that does not solve any actual problem in this particular case, because we generally are not interested in *extremely small* strains :)

Reply:
just a good habit, nothing more ;-)

Reply:
Ok. I'll brush my teeth twice, just to make sure!